### DISCUSSION TITLE
Endpoint for retrieving the current season name

### DISCUSSION THREAD

**Discussion by PyFlat-JR** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#301] [Q&A] [answered]
Hi, so far I have used this endpoint to get the name of the current season: 

https://fortnite.com/en-US/api/battle-pass-data --> (slug: chapter5-season3)

However, it seems to be outdated now, as it has not been updated this season and still shows the old season. 

Is there another way to get the name of the current season with the API?

**Comment by LeleDerGrasshalmi** [comment]
epic recently migrated the battle pass page to remix, you gotta use that page's preload data:

`https://www.fortnite.com/battle-pass?lang=en-US&_data=routes%2Fbattle-pass._index`

but ye, the site has cloudflare bot protection ._.

**Reply by PyFlat-JR** [reply]
ok, it also seems that it doesnt contains the season name anymore either. Is there any other known endpoint to get the season name?

**Reply by LeleDerGrasshalmi** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/0db8c34c-e33b-486c-a6a7-8c7cd785b857)

not as a plain string, but you can extract it ig

**Reply by PyFlat-JR** [reply]
no i don't mean the name itself i mean the idtentifier like chapter 5 season 4. I apologize if I did not make this clear properly.
